test/fixtures/works.yml

```diff
@@ -0,0 +1,76 @@
+work_one:
```

Comment:
Fixtures look great, but they're not used in your tests. Please let me know if you have questions about how to use fixtures.